models/checkpoint

```diff
@@ -1,2 +1,2 @@
-model_checkpoint_path: "dcscn_L2_F4to4_PS_R1F4.ckpt"
-all_model_checkpoint_paths: "dcscn_L2_F4to4_PS_R1F4.ckpt"
+model_checkpoint_path: "inference_model.ckpt"
```

Comment:
It looks this file is committed by mistake. Actually we should remove this file from the repository.

Reply:
Also please exclude those private model files (.ckpt.*)
If we the model looks good and you want to include the model, I would like to check the performance. ;)